server/data.py

```diff
@@ -11,8 +11,8 @@
 class Datafetch(object):
 
     def _init_tabix(self):
-        self.tabix_files_imputed = defaultdict(lambda: [pysam.TabixFile(file, parser=None) for file in self.conf['vcf_files']['imputed_data']])
-        self.tabix_files_chip = defaultdict(lambda: [pysam.TabixFile(file, parser=None) for file in self.conf['vcf_files']['chip_data']])
+        self.vcfs_imputed = defaultdict(lambda: [file for file in self.conf['vcf_files']['imputed_data']])
```

Comment:
We don't need to dict here anymore - self.vcfs_imputed can be directly the list of VCFs

The dict was used for the tabix handles because of several worker threads - each thread should only use its own tabix handle. But the VCF filenames are the same for all threads so we don't need a map from a thread to VCFs

Reply:
Right, no need in this indeed! I updated the code.